ros/src/twist_controller/twist_controller_mod.py

```diff
@@ -34,6 +35,9 @@ def __init__(self, default_update_interval, wheel_base, steer_ratio, min_speed,
         self.dynamic_reconf_server = Server(
             DynReconfConfig, self.handle_dynamic_variable_update)
 
+        self.throttle_lpf = LowPassFilter(0.5, default_update_interval)
```

Comment:
The problem with this approach is that I'm not sure how to pick an appropriate value for `tau`, given that I haven't seen the PID controllers respond with large values quickly.

One way to test would be to artificially replace the PID controller code with wildly varying actuations to verify that things work as expected.

Perhaps I should just use a constant limit ex: `max(throttle_limit, self.velocity_pid_controller.step(...))`? That wouldn't require tuning the LPF

Reply:
One way to tune tau is to slow it down until the output starts to get restricted in normal operation, then back it off a bit.  So far, the biggest step change for throttle I see is when initially clicking the manual box to OFF so the controller has gets a step input to the initial target speed.  It’s still not that big so we could force some bigger input artificially but for now I think just tuning it around the edge of normal operation is enough to start.